compose/docker-compose-local.yml

```diff
@@ -1,41 +1,66 @@
-version: '3'
-name: request-logger
-
-services:
-    web:
-        image: nginx
-        container_name: nginx
-        volumes:
-            - ../src/nginx.conf:/etc/nginx/nginx.conf:ro
-        command: [nginx-debug, '-g', 'daemon off;']
-        ports:
-            - "8080:80"
-        environment:
-            - NGINX_HOST=foobar.com
-            - NGINX_PORT=80
-        networks:
-            - nginx.docker
-    
-    requestlogger:
-        container_name: request-logger
-        build:
-            context: ../src
-            dockerfile: RequestLogger\Dockerfile
-        networks:
-            - nginx.docker
-            
-    mockserver:
-      image: mockserver/mockserver:latest
-      container_name: mock-server
-      environment:
-        MOCKSERVER_WATCH_INITIALIZATION_JSON: "true"
-        MOCKSERVER_PROPERTY_FILE: /config/mockserver.properties
-        MOCKSERVER_INITIALIZATION_JSON_PATH: /config/initializerJson.json
-      volumes:
-        - ../src/MockServerConfig/initializerJson.json:/config/initializerJson.json:ro
-      networks:
-        - nginx.docker
-
-networks:
-  nginx.docker:
-    name: nginx.docker
\ No newline at end of file
+version: '3'

```

Comment:
Is it worth renaming this `docker-compose.yml` to make running as `docker compose up` easier?

Reply:
I left it as this in case more compose files were added to make clear what this one was for, but I'm not sure there will be, so I'll change it